lib/util/AuthConfig.js

```diff
@@ -111,9 +120,64 @@ class AuthConfig extends Config {
             this.loginKey = undefined;
             this.refreshToken = undefined;
         }
+
+        var tempJSON = this._getAuthJSON();
+        
+
+        if(tempJSON == undefined) {
+            tempJSON = {};
+        }
+
+        if(Object.values(this._json)[0].isDefault) {
```

Comment:
Not sure the logic of this if and loop are quite right? Based on what we reviewed together at your machine, has this code been updated since you pushed?

```javascript
var defaultAccountID;
for (var account in tempJSON) {
                if(tempJSON[account].isDefault){
                    defaultAccountID = account;
                }
}
```

Reply:
I may be just understanding this block of code but `Object.keys(...)[0]` make me nervous without seeing a sort added somewhere :)

Reply:
[0]Yeah it's working fine, so this just makes sure that the user chose the new logged in account to be a default one, and if so, then we delete the default from the old default one.

And for the `object.values(this._json)[0]` there is only to be one value in `this._json` so it's not a problem here.